internal/access/identity.go

```diff
@@ -15,19 +15,24 @@ import (
 )
 
 // isIdentitySelf is used by authorization checks to see if the calling identity is requesting their own attributes
-func isIdentitySelf(rCtx RequestContext, userID uid.ID) (bool, error) {
+func isIdentitySelf(rCtx RequestContext, userID uid.ID) bool {
 	identity := rCtx.Authenticated.User
-	return identity != nil && identity.ID == userID, nil
+	return identity != nil && identity.ID == userID
 }
 
 func GetIdentity(c *gin.Context, id uid.ID) (*models.Identity, error) {
-	roles := []string{models.InfraAdminRole, models.InfraViewRole, models.InfraConnectorRole}
-	db, err := hasAuthorization(c, id, isIdentitySelf, roles...)
-	if err != nil {
-		return nil, HandleAuthErr(err, "user", "get", roles...)
+	rCtx := GetRequestContext(c)
+	if isIdentitySelf(rCtx, id) {
```

Comment:
Should this match the same pattern as `credentials.go`? It's cleaner to me.
```
if !isSelf {
		err := IsAuthorized(rCtx, models.InfraAdminRole)
		if err != nil {
			return HandleAuthErr(err, "user", "update", models.InfraAdminRole)
		}
	}
```

Reply:
Ya, I like that. I think in the other places we have this we have the empty `if` block, but we can fix those as we find them

Reply:
Done!